highest_number_cubed/main.py

```diff
@@ -1,6 +1,10 @@
 def highest_number_cubed(limit):
-    pass
-
+    for i in range(1,limit):
+        answer = i**3
+        if answer <= limit:
+            continue
+        else:
+            return (i - 1)
```

Comment:
I like how you're making use of index numbers to get the highest number. 
An alternative solution would be to create list of all the numbers using the function `.append()` 
and then return the last number on the list with `[-1]`